components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -210,7 +238,17 @@ subroutine modal_aer_opt_init()
 
    ! Add diagnostic fields to history output.
 
+   call addfld ('MODAL_AER_TAU_SW', (/'lev   ','swband'/), 'A', '1', &
```

Comment:
Can you see if there's a CF standard name for any of these new variables and add it to the addfld call?
https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html

Reply:
I did some searching and looks like aerosol-radiative properties broken down by short/longwave are not there.

Reply:
Use the standard_name optional argument at the end of the call.
```
call addfld ('MODAL_AER_TAU_SW', (/'lev ','swband'/), 'A', '1', &
'Aerosol shortwave extinction optical depth', flag_xyfill=.true., &
standard_name='atmosphere_optical_thickness_due_to_ambient_aerosol_particles')
```